spec/support/xml_attribute_block_matcher.rb

```diff
@@ -18,11 +18,11 @@
 
 RSpec::Matchers.define :have_response_value do |expected|
   match do |actual|
-    actual_response_type = actual.css('ResponseValue').text
-    actual_response_type == expected
+    actual_response_value = actual.css('ResponseValue').text
+    actual_response_value == expected
```

Comment:
My guess is the problem is trailing white space. What happens if you change this to:
```ruby
actual_response_value.strip == expected.strip
```

Reply:
Also can you post an example of the failure message.